### PR TITLE
OTG: Making the test RT-3.2 align with ATE test

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
Log:
[RT_3.2.docx](https://github.com/openconfig/featureprofiles/files/10792266/RT_3.2.docx)

### CODE REVIEW

File: feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ var (
 
 	atePort2 = attrs.Attributes{
 		Name:    "atePort2",
-		MAC:     "01:00:02:01:01:01",
+		MAC:     "00:12:01:00:00:01",
```
Comment (michaelore) [inline feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go:81]: I'm curious why you're changing the MAC addresses here? I'm not sure what effect it has on the test

**Review by michaelore — CHANGES_REQUESTED** [review]
Should we add a check for *deviations.NoMixOfTaggedAndUntaggedSubinterfaces as well, as in the ATE test? Not sure if it's in scope or there's any action to take.

This line in the ATE test: https://github.com/openconfig/featureprofiles/blob/e97253d3dcf6eea36fee982e3344ec00148406e8/feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go#L151

File: feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go (hunk 1 above)
Comment (ANISH-GOTTAPU) [reply feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go:81]: sorry for late reply, I was on PTO

cptx(juniper) is not resolving ARP with mac address `01:00:02:01:01:01` on ixia-c , I'm checking internally and with juniper team on this and get back to you. 

image

**Comment by ANISH-GOTTAPU** [comment]
> Should we add a check for *deviations.NoMixOfTaggedAndUntaggedSubinterfaces as well, as in the ATE test? Not sure if it's in scope or there's any action to take.
> 
> This line in the ATE test:
> 
> https://github.com/openconfig/featureprofiles/blob/e97253d3dcf6eea36fee982e3344ec00148406e8/feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go#L151

Added this check

**Review by michaelore — APPROVED** [review]

**Pull request merged**